onyo/main.py

```diff
@@ -212,7 +223,7 @@ def setup_parser() -> argparse.ArgumentParser:
         'set',
         description=commands.set.__doc__,
         formatter_class=SubcommandHelpFormatter,
-        help=commands.set.__doc__
+        help='Set the value of keys for matching assets.'
```

Comment:
"for matching assets" is IMO missleading. `onyo set --match` does not exist anymore, it must be done via piping, xargs etc, like discussed yesterday, or simply via giving the paths to assets you want to change.

The sentence works perfectly without the word "matching" :)

Reply:
I was leaving those until after Ben's PR is merged. But I can change those now.

Reply:
Fixed and pushed.